step.yml

```diff
@@ -75,7 +81,6 @@ inputs:
     value_options:
     - "true"
     - "false"
-    is_dont_change_value: true
```

Comment:
I think it's perfectly fine to change this input, especially in persistent self-hosted envs